Mlem/App/Views/Root/Tabs/Settings/FiltersSettingsView.swift

```diff
@@ -23,9 +25,13 @@ struct FiltersSettingsView: View {
     var body: some View {
         List {
             Section {
-                keywordSection
+                Toggle("Enable Keyword Filters", isOn: $keywordFilterEnabled)
             } header: {
                 Text("Keyword Filters")
+            }
```

Comment:
I don't think we need this header anymore

```suggestion
            }
```